build/fileindex.js

```diff
@@ -4,35 +4,33 @@ module.exports = {
             format: 'json_flat',
             pretty: true
         },
-        files: [{ 
-            dest: global.dist + '/javascript.json', 
+        files: [{
+            dest: global.dist + '/javascript.json',
             src: [
-            'src/javascript/lib/jquery.js', 
-            'src/javascript/lib/highstock/highstock.js', 
+            'src/javascript/lib/jquery.js',
+            'src/javascript/lib/highstock/highstock.js',
             'src/javascript/lib/highstock/highstock-exporting.js',
-            'src/javascript/lib/moment/moment.js', 
+            'src/javascript/lib/moment/moment.js',
             'src/javascript/lib/**/*.js',
```

Comment:
let's leave it for now @eugene-eeo, I don't want this PR too get too scary :) we can do it in further refactoring, as there is a lot that can be done to optimize the builds and grunt tasks.